changelog/issue-7979.md

```diff
@@ -0,0 +1,7 @@
+audience: worker-deployers
+level: major
```

Comment:
```suggestion
level: minor
```

I _think_ this isn't breaking, should be safe at `minor` level, but please correct me if I'm wrong!